config/locales/views.de.yml

```
@@ -525,6 +525,8 @@ de:
       participants: 'Teilnehmende'
     export_enqueued: 'Export wird im Hintergrund gestartet und nach Fertigstellung heruntergeladen.'
     add_tag: Tag hinzufügen
+    invitations:
+      description: 'Eingeladene Personen können sich via %{model_name}-Seite für den %{model_name} an- oder abmelden. Sie erhalten kein automatisches Mail.'
```

```suggestion
      description: 'Eingeladene Personen können sich via %{model_name}-Seite an- oder abmelden. Sie erhalten kein automatisches Mail.'
```
So stimmt die Description auch im Fall von Lagern, und verliert trotzdem nur wenig an Klarheit.